### PR TITLE
Update work-with-tasks.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
- Context for task inbox.

### CODE REVIEW

File: source/playbooks/work-with-tasks.rst [hunk 1]
```diff
@@ -8,13 +8,13 @@ In some workflows, there are time constraints on tasks and others may have more
 
 To assign a due date to a task, select the **Toggle Run Details** icon to open the **Run Details** screen. Hover over the task you’d like to edit and select the calendar icon to assign a due date. Due dates can be used to sort tasks in the run overview.
 
-When a due date is assigned to a task, and the task is overdue or due today, a reminder is added to the Playbooks daily digest along with tasks that don't have an assigned due date. As tasks are completed, they're removed from the daily digest reminders. You can refresh the list of assigned tasks at any time using the ``/playbooks to-do`` slash command.
+When a due date is assigned to a task, and the task is overdue or due today, a reminder is added to the Playbooks daily digest along with tasks that don't have an assigned due date. As tasks are completed, they're removed from the daily digest reminders. You can refresh the list of assigned tasks at any time using the ``/playbooks todo`` slash command.
```
Comment (trilopin) [inline source/playbooks/work-with-tasks.rst:11]:
```suggestion
When a due date is assigned to a task, and the task is overdue or due today, a reminder is added to the Playbooks daily digest along with tasks that don't have an assigned due date. As tasks are completed, they're removed from the daily digest reminders. You can refresh the list of assigned tasks at any time using the ``/playbook todo`` slash command.
```

slash command fixed, it's always singular

File: source/playbooks/work-with-tasks.rst [hunk 2]
```diff
@@ -8,13 +8,13 @@ In some workflows, there are time constraints on tasks and others may have more
...
 Due dates can be entered in text (e.g., "two minutes ago") or numerically (e.g., "15 March 2023").
 
 Task inbox
 ----------
 
-In addition to the daily digest, you also have access to a task inbox. To view it, open the playbooks tab. 
+In addition to the daily digest, you also have access to a task inbox. The task inbox provides you with a cross-run overview of the tasks you're accountable for, sorted by due date. You can access and complete each task directly, without having to visit the individual runs.
 
-In the header, next to your profile photo, select the message icon. A list of every task assigned to you from every run that's in progress is displayed. You can check off, change due dates, and reassign tasks in this view.
+To view it, open the playbooks tab. In the header, next to your profile photo, select the message icon. A list of every task assigned to you from every run that's in progress is displayed. You can check off, change due dates, and reassign tasks in this view.
```
Comment (trilopin) [inline source/playbooks/work-with-tasks.rst:20]: nit: it's not a message icon but a tasks list icon. TBF it's super tiny and easy to misunderstand

File: source/playbooks/work-with-tasks.rst [hunk 3]
```diff
@@ -8,13 +8,13 @@ In some workflows, there are time constraints on tasks and others may have more
...
+When a due date is assigned to a task, and the task is overdue or due today, a reminder is added to the Playbooks daily digest along with tasks that don't have an assigned due date. As tasks are completed, they're removed from the daily digest reminders. You can refresh the list of assigned tasks at any time using the ``/playbooks todo`` slash command.
 
 Due dates can be entered in text (e.g., "two minutes ago") or numerically (e.g., "15 March 2023").
 
 Task inbox
 ----------
 
-In addition to the daily digest, you also have access to a task inbox. To view it, open the playbooks tab. 
+In addition to the daily digest, you also have access to a task inbox. The task inbox provides you with a cross-run overview of the tasks you're accountable for, sorted by due date. You can access and complete each task directly, without having to visit the individual runs.
```
Comment (trilopin) [inline source/playbooks/work-with-tasks.rst:18]: not sure if it's a good idea to make this more complex, but there are more interesting things that the user could do with each task in the context of task inbox:
- change assignee to change accountability -> disappear from task inbox
- change duedate -> change urgency

What I'm trying to explain is, if this resource mix "what I'm accountable for" + "sense of urgency" the main actions are:
- finish/skip the task by myself
- assign to another person 
- change the sense of urgency

**Review by trilopin — CHANGES_REQUESTED** [review]
slash command needs to be fixed, the rest of the comments are optional or just thoughts

File: source/playbooks/work-with-tasks.rst [hunk 4]
```diff
@@ -8,13 +8,13 @@ In some workflows, there are time constraints on tasks and others may have more
...
 
-In the header, next to your profile photo, select the message icon. A list of every task assigned to you from every run that's in progress is displayed. You can check off, change due dates, and reassign tasks in this view.
+To view it, open the playbooks tab. In the header, next to your profile photo, select the message icon. A list of every task assigned to you from every run that's in progress is displayed. You can check off, change due dates, and reassign tasks in this view.
```
Comment (justinegeffen) [inline source/playbooks/work-with-tasks.rst:20]:
```suggestion
To view it, open the playbooks tab. In the header, next to your profile photo, select the tasks list icon. A list of every task assigned to you from every run that's in progress is displayed. You can check off, change due dates, and reassign tasks in this view.
```

File: source/playbooks/work-with-tasks.rst (hunk 3 above)
Comment (justinegeffen) [reply source/playbooks/work-with-tasks.rst]: Great suggestions, thank you!

File: source/playbooks/work-with-tasks.rst [hunk 5]
```diff
@@ -8,13 +8,20 @@ In some workflows, there are time constraints on tasks and others may have more
...
 Task inbox
 ----------
 
-In addition to the daily digest, you also have access to a task inbox. To view it, open the playbooks tab. 
+In addition to the daily digest, you also have access to a task inbox. The task inbox provides you with a cross-run overview of the tasks you're accountable for, sorted by due date.
 
-In the header, next to your profile photo, select the message icon. A list of every task assigned to you from every run that's in progress is displayed. You can check off, change due dates, and reassign tasks in this view.
+You can:
+
+- You can access each task directly, without having to visit the individual runs.
```
Comment (trilopin) [inline source/playbooks/work-with-tasks.rst:22]:
```suggestion
- Access each task directly, without having to visit the individual runs.
```

File: source/playbooks/work-with-tasks.rst [hunk 6]
```diff
@@ -8,13 +8,20 @@ In some workflows, there are time constraints on tasks and others may have more
...
+- You can mark tasks as complete or skip them.
```
Comment (trilopin) [inline source/playbooks/work-with-tasks.rst:23]:
```suggestion
- Mark tasks as complete or skip them.
```

**Review by trilopin — APPROVED** [review]
Thanks 🎉

**Comment by justinegeffen** [comment]
> Thanks 🎉

Thank YOU for the awesome feedback! :)

**Review by cwarnermm — APPROVED** [review]
Great update focused on the user's success!

**Pull request merged**